Formula/smap.rb

```diff
@@ -0,0 +1,25 @@
+class Smap < Formula
+  desc "Is a drop-in replacement for Nmap powered by shodan.io"
```

Comment:
Based on the format that we use for most formulae, I'd suggest removing the "is a" at the beginning:
```ruby
  desc "Drop-in replacement for Nmap powered by shodan.io"
```

Reply:
You are are right. Done